_maps/_basemap.dm

```diff
@@ -12,7 +12,9 @@
 		#include "map_files\MetaStation\MetaStation.dmm"
 		#include "map_files\IceBoxStation\IceBoxStation.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
-
+		#include "map_files\Blueshift\BlueShift_lower.dmm"
```

Comment:
Are these all supposed to be defined here? It isn't like this for Icebox.

Reply:
i believe ice box's separate levels not being here is an oversight

Reply:
as in, we used to have the z-levels on the icebox dmm itself but we split them recently without updating it here

Reply:
it isnt in the .dmm  but I believe is in the .json

Reply:
gamer, this is the "compile all maps" CI task
every dmm file should be in there